Source/Routing and Faulting/Routing and Faulting Custom Device.lvproj

```diff
@@ -682,19 +675,19 @@
 				<Property Name="Bld_excludedDirectory[5]" Type="Path">user.lib</Property>
 				<Property Name="Bld_excludedDirectory[5].pathType" Type="Str">relativeToAppDir</Property>
 				<Property Name="Bld_excludedDirectoryCount" Type="Int">6</Property>
-				<Property Name="Bld_localDestDir" Type="Path">../Built/Routing and Faulting/Routing and Faulting Scripting API.llb</Property>
+				<Property Name="Bld_localDestDir" Type="Path">/C/dev/CustomDevice/niveristand-routing-and-faulting-custom-device/Built/Routing and Faulting/Windows/Routing and Faulting Scripting API.llb</Property>
 				<Property Name="Bld_previewCacheID" Type="Str">{4DD6684D-3970-4A6D-A0E8-84ECE828ED70}</Property>
-				<Property Name="Bld_version.build" Type="Int">5</Property>
+				<Property Name="Bld_version.build" Type="Int">6</Property>
 				<Property Name="Bld_version.major" Type="Int">1</Property>
 				<Property Name="Destination[0].destName" Type="Str">Destination Directory</Property>
-				<Property Name="Destination[0].path" Type="Path">../Built/Routing and Faulting/Routing and Faulting Scripting API.llb</Property>
+				<Property Name="Destination[0].path" Type="Path">/C/dev/CustomDevice/niveristand-routing-and-faulting-custom-device/Built/Routing and Faulting/Windows/Routing and Faulting Scripting API.llb</Property>
 				<Property Name="Destination[0].path.type" Type="Str">&lt;none&gt;</Property>
 				<Property Name="Destination[0].type" Type="Str">LLB</Property>
 				<Property Name="Destination[1].destName" Type="Str">Support Directory</Property>
-				<Property Name="Destination[1].path" Type="Path">../Built/Routing and Faulting/data</Property>
+				<Property Name="Destination[1].path" Type="Path">/C/dev/CustomDevice/niveristand-routing-and-faulting-custom-device/Built/Routing and Faulting/Windows/data</Property>
```

Comment:
The Scripting Build Spec still has absolute path. Update the destination directories to match other build specifications.